app/navigation/Navigator.scala

```diff
@@ -327,11 +328,16 @@ class Navigator @Inject()(implicit frontendAppConfig: FrontendAppConfig)
   }
 
   private[this] def employeeRTISubmissionRoutes: UserAnswers => Call = { userAnswers =>
-    userAnswers.getV(EmployeeRTISubmissionPage) match {
-      case Valid(Yes)                                         => handlePayDateRoutes(userAnswers)
-      case Valid(No) if isEnabled(ExtensionTwoNewStarterFlow) => routes.OnPayrollBefore30thOct2020Controller.onPageLoad()
-      case Valid(No)                                          => routeToEmployeeFirstFurloughed(userAnswers)
-
+    (userAnswers.getV(EmployeeRTISubmissionPage), userAnswers.getV(EmployeeStartDatePage)) match {
```

Comment:
So this one should check if the claim period start date is before the 1st November. Because it should only kick them out if they say No and they're trying to do a classic claim.